mkt/webpay/webpay_jwt.py

```diff
@@ -77,6 +87,20 @@ def external_id(self):
     def name(self):
         return self.webapp.name
 
+    def default_locale(self):
+        return self.webapp.default_locale
+
+    def localized_properties(self):
+        props = {}
+
+        for attr in ('name', 'description'):
+            tr_object = getattr(self.webapp, attr)
+            for tr in Translation.objects.filter(id=tr_object.id):
```

Comment:
Is there no way to retrieve this from `tr_object` directly? This seems fragile if we add support for activating/deactivating locales for example.